core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.component.html

```
@@ -42,6 +39,23 @@
         </div>
+        <div class="question-pagination-box" *ngIf="questionsFetched && totalQuestionCount > 0">
+          <span class="question-pagination-display"
+            [innerHTML]="'I18N_SKILL_PREVIEW_TAB_QUESTIONS_NUMBER_DISPLAY' | translate: { startingNumber: firstQuestionOnPageNum, endingNumber: lastQuestionOnPageNum, totalNumber: totalQuestionCount }">
+          </span>
+          <ngb-pagination [collectionSize]="totalQuestionCount" [(page)]="page" [maxSize]="2"
+            [pageSize]="QUESTION_COUNT" [rotate]="true" (pageChange)="onPageChange($event)" class="ms-auto me-auto">
+            <ng-template ngbPaginationPrevious>
+              <i class="fa fa-angle-left text-center w-100" aria-hidden="true"></i>
+            </ng-template>
+            <ng-template ngbPaginationNumber let-p>
+              <button class="pagination-button">{{ p }}</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)